themes/origins/snippets/yc-icon.liquid

```diff
@@ -0,0 +1,81 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  fill="none"
+  viewBox="0 0 24 24"
+  stroke-width="1.5"
+  stroke="currentColor"
+  height="100%"
+  width="100%"
```

Comment:
what do u think about making this snippet accept optional props for `width` and `height`? based on our _demo_, I think it would be a useful feature. 
_Default size: `18px`_